src/components/About/Blurbs.js

```diff
@@ -65,9 +65,9 @@ class Blurbs extends Component {
 
         </div>
 
-        <div style = {{padding: "20px 0px 20px 0px"}}>
+        <div className = "blurbDivs">
 
-          <h3 className = "LeftBlurbs" style = {{fontSize: "22pt"}}>
+          <h3 className = "Blurbs" style = {{fontSize: "22pt"}}>
```

Comment:
While the Blurbs.js works with most of the styling moved to About.css, it would also be good practice if `<img className = "LeftBlurbImages" src={require("../../img/img-clipart-os-coffee.png")} style = {{backgroundColor: "white"}}/ >` and `<h3 className = "Blurbs" style = {{fontSize: "22pt"}}>` are moved to About.css .